addons/account/views/report_invoice.xml

```diff
@@ -225,7 +222,8 @@
             <t t-call="web.html_container">
                 <t t-foreach="docs" t-as="o">
                     <t t-set="lang" t-value="o.invoice_user_id.sudo().lang if o.move_type in ('in_invoice', 'in_refund') else o.partner_id.lang"/>
-                    <t t-call="account.report_invoice_document_with_payments" t-lang="lang"/>
+                    <t t-set="print_with_payments" t-value="True"/>
+                    <t t-call="#{ o._get_custom_report('account.report_invoice_document') }" t-lang="lang"/>
```

Comment:
@zaoral We prefer not to pass the name of the report here, but _get_invoice_report(show_payments=False) as header instead.

Reply:
Hi @jco-odoo . Great, I removed the name of the report as parameter and return the name as the default one inside the method _get_custom_report(). You can see the change in this commit https://github.com/odoo/odoo/pull/54193/commits/94eccccc1ea8e331e19d5c39708da252ebdba82c

However I want to ask you, I did not see why you want to add show_payments=False in the method? Please let me know